content/en/synthetics/browser_tests/test_runs.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Browser Test Runs
```

Comment:
Talking to @cosme-ddog about this, and we think we should maybe continue to call this page `Browser Test Results`, as that is predominately what this page is focusing on (including 'Sample Results', 'Test Results table etc). What do you think?